src/Nowruz/modules/general/components/SearchDropdown/index.tsx

```diff
@@ -50,6 +50,14 @@ const CustomSingleValue = (props: any) => {
     </components.SingleValue>
   );
 };
+
+const CustomClearIndicator = (props: ClearIndicatorProps) => {
+  return (
+    <components.ClearIndicator {...props}>
+      <Icon name="x-close" fontSize={20} color="#667085" className="cursor-pointer" />
+    </components.ClearIndicator>
```

Comment:
I think it just need to remove padding of close indicator to be similar to dropdown indicator position

Reply:
no need to remove padding, we should handle the condition for `hasDropdownIcon` properly